sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateIssuer.cs

```diff
@@ -185,7 +185,6 @@ private void ReadOrgDetailsProperties(JsonElement json)
                             admin.ReadProperties(elem);
                             AdministratorContacts.Add(admin);
                         }
-                        Password = prop.Value.GetString();
```

Comment:
While the service will probably never return this and this change makes sense at face value, we don't want to ignore any value returned from the server. Instead, I would check first if this value is present and, if so, deserialize as a string (we want to throw if the type is unexpected only).

Reply:
We already do this [here](https://github.com/Azure/azure-sdk-for-net/blob/20908895b77715fb7d00699e8df42ae43d8f2b27/sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateIssuer.cs#L164-L166)